book/src/user_manual/credential_validation.md

```diff
@@ -0,0 +1,12 @@
+# Credential validation
+
+Credential validation is a process that allows a member to verify the validity
+of the credentials of other members in the group.  The process is described in
+detail in the [MLS protocol
+specification](https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-credential-validation).
```

Comment:
We should probably link to the datatracker version: https://www.ietf.org/archive/id/draft-ietf-mls-protocol-20.html#name-credential-validation

The MLS spec already refers to the architecture doc, but it might be helpful to mention the AS specifically and include the following link: https://www.ietf.org/archive/id/draft-ietf-mls-architecture-10.html#name-authentication-service